test/integration/aws/default/build/flow_log.tf

```diff
@@ -0,0 +1,68 @@
+resource "aws_flow_log" "fl-vpc" {
+  log_group_name = "${aws_cloudwatch_log_group.test_log_group.name}"
+  iam_role_arn   = "${aws_iam_role.test_role.arn}"
+  vpc_id         = "${data.aws_vpc.default.id}"
+  traffic_type   = "ALL"
+}
+
+resource "aws_flow_log" "fl-subnet" {
+  log_group_name = "${aws_cloudwatch_log_group.test_log_group.name}"
+  iam_role_arn   = "${aws_iam_role.test_role.arn}"
+  subnet_id      = "${aws_subnet.subnet_01.id}"
+  traffic_type   = "ALL"
+}
+
+output "fl-vpc" {
+  value = "${aws_flow_log.fl-vpc.id}"
+}
+
+output "fl-subnet" {
+  value = "${aws_flow_log.fl-subnet.id}"
+}
+resource "aws_cloudwatch_log_group" "test_log_group" {
+  name = "test_log_group"
+}
+
+resource "aws_iam_role" "test_role" {
```

Comment:
Same - could we get a name that is specific to flow log testing?

Terraform's namespace is flat, so if anyone ever uses that name, we have a collision.